src/argilla/training/openai.py

```diff
@@ -14,11 +14,12 @@
 
 import logging
 import os
+import time
 from typing import List, Union
 
 import numpy as np
 
-from argilla._constants import OPENAI_END_TOKEN, OPENAI_SEPARATOR, OPENAI_WHITESPACE
+from argilla._constants import OPENAI_END_TOKEN, OPENAI_LEGACY_MODEL, OPENAI_SEPARATOR, OPENAI_WHITESPACE
```

Comment:
If the previous suggestion on the rename of `OPENAI_LEGACY_MODEL` TO `OPENAI_LEGACY_MODELS` was applied, we should also rename it here

```suggestion
from argilla._constants import OPENAI_END_TOKEN, OPENAI_LEGACY_MODELS, OPENAI_SEPARATOR, OPENAI_WHITESPACE
```